metatron-app/metatron-discovery
# Fn fix exploredata error

### Description

1. An error occurs when [Management] Metadata Details> Go to Data Source> Delete Data Source.
2. [Management] Data search> Searchability on / off does not work when generating metadata.
3. In the [Management] Data Navigation> Code Table> Search Result List screen, the delete icon UI overlaps.
4. The [Management] metadata table list is different from the plan.
5. In the [Management] Data Navigation> Glossary> Glossary Registration popup, after entering the glossary name, the validity check is not performed properly when focusing out.
6. Fix when click some data can't go to detail view in explore data
7. fix resize component when lnb is folded in explore data

**Related Issue** : 

METATRON-3020
METATRON-3003
METATRON-2990
METATRON-2955
METATRON-2989
METATRON-2971
METATRON-3023
#2501 

### How Has This Been Tested?

2971
1. Explore Data 
![image](https://user-images.githubusercontent.com/35441780/64226267-a9b73900-cf19-11e9-80fa-a709062b7360.png)
Check especially  when click data right above, detail pop up is showing 

3023
1. Explore Data
Check The main banner's text and tag ellipsis is working

2501
1. Explore data
check when fold lnb explore component is resize correctly

1. Go to [Management] Metadata Details> Data Sources> Verify that the data source is deleted correctly.
2. Check if [Management] data exploration> metadata creation is turned off or disabled.
3. On the [Management] Data Navigation> Code Table> Search Result List screen, confirm that the Delete icon UI is displayed correctly.
4. Confirm that the [Management] metadata table list form is displayed as planned.
5. In the [Management] Data Navigation> Glossary> Glossary Registration popup, enter the glossary name and check if the validity check works properly when focusing out.

#### Need additional checks?

### Types of changes



### Checklist:


### Additional Context
